optimizely/decision_service.py

```diff
@@ -517,11 +450,37 @@ def get_variation_for_feature(self, project_config, feature, user_context, optio
             # Evaluate each experiment ID and return the first bucketed experiment variation
             for experiment in feature.experimentIds:
                 experiment = project_config.get_experiment_from_id(experiment)
-                if experiment:
-                    variation, variation_reasons = self.get_variation_from_experiment_rule(
-                        project_config, feature.key, experiment, user_context, options)
-                    decide_reasons += variation_reasons
-                    if variation:
-                        return Decision(experiment, variation, enums.DecisionSources.FEATURE_TEST), decide_reasons
+                decision_variation = None
 
+                if experiment:
+                    # variation, variation_reasons = self.get_variation_from_experiment_rule(
```

Comment:
Remove this commented code

Reply:
I was testing this code with travis. Will remove it on finalization.